spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -2053,6 +2061,126 @@ def setup_firefox_with_linux
         end
         it_behaves_like 'default case'
       end
+
+      context "id=collect_logs" do
+        context "server is not started" do
+          it "disables button with correct title" do
+            @record.status = "not responding"
+            error_msg = "Cannot collect current logs unless the Server is started"
+            expect(build_toolbar_disable_button("collect_logs")).to eq(error_msg)
+          end
+        end
+
+        context "server is started" do
+          context "log collecting is in progress, log_file.state = collection" do
+            it "disables button with correct title" do
```

Comment:
We have unneeded contexts... contexts with a single it could be collapsed here.  Try to make the it description shorter so you don't need the context, something like `it "'collecting' log_file with started server"`?